### ISSUE TITLE
-[SDWebImageImageIOCoder decodedImageWithData:] (SDWebImageImageIOCoder.m:106) Crash

### ISSUE THREAD

**Issue by Aoce** [SDWebImage/SDWebImage#3117]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS
 Platform Version        | e.g. 9.3.2 / 12.4.7 / 13.6  / 14.0.1 / 
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

ColorSync | ColorSyncProfileCopyLocalizedDescriptionString + 328
-- | --
1 ColorSync | ColorSyncProfileCopyLocalizedDescriptionString + 324
2 ColorSync | ColorSyncProfileCopyDescriptionString + 44
3 CoreGraphics | _icc_create_icc_profile_description + 80
4 ImageIO | IIOImagePlus::setGeometry(void*, CGImagePlusGeometry) + 500
5 ImageIO | IIOReadPlugin::prepareGeometry(InfoRec*) + 784
6 ImageIO | IIOReadPlugin::setupImageProviderCallbackV2(InfoRec*, CGImageBlockSet* (*)(void*, CGImageProvider*, CGRect, CGSize, __CFDictionary const*), __IOSurface* (*)(void*, CGImageProvider*, __CFDictionary const*)) + 32
7 ImageIO | IIOReadPlugin::callInitialize() + 900
8 ImageIO | IIO_Reader_AppleJPEG::initImageAtOffset(CGImagePlugin*, unsigned long, unsigned long, unsigned long) + 164
9 ImageIO | IIOImageSource::makeImagePlus(unsigned long, IIODictionary*) + 968
10 ImageIO | IIOImageSource::createImageAtIndex(unsigned long, IIODictionary*) + 80
11 ImageIO | CGImageSourceCreateImageAtIndex + 244
12 UIKitCore | __UIImageRefFromData + 292
13 UIKitCore | -[UIImage _initWithData:preserveScale:cache:scale:] + 60
14 XXXXX | -[SDWebImageImageIOCoder decodedImageWithData:] (SDWebImageImageIOCoder.m:106)
15 XXXXX | -[SDWebImageCodersManager decodedImageWithData:] (SDWebImageCodersManager.m:108)
16 XXXXX | __70-[SDWebImageDownloaderOperation URLSession:task:didCompleteWithError:]_block_invoke.264 (SDWebImageDownloaderOperation.m:0)
17 libdispatch.dylib | __dispatch_call_block_and_release + 24
18 libdispatch.dylib | __dispatch_client_callout + 16
19 libdispatch.dylib | __dispatch_lane_serial_drain$VARIANT$armv81 + 564
20 libdispatch.dylib | __dispatch_lane_invoke$VARIANT$armv81 + 396
21 libdispatch.dylib | __dispatch_workloop_worker_thread + 580
22 libsystem_pthread.dylib | _pthread_wqthread + 272

**Comment by dreampiggy** [comment]
Any detail crash information ? Like the C assert, or debug information. This stacktrace only provide the caller API `UIImage imageWithData:`, but no any detail information.

Or, any image URL can re-produce this issue ?
